buildSrc/src/main/groovy/Versions.groovy

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright (c) 2019 Connexta, LLC
+
+Released under the GNU Lesser General Public License version 3; see
+https://www.gnu.org/licenses/lgpl-3.0.html
+*/
+// Default Package
+class Versions {
+    static String java = "1.11"
+    static String project = "0.1.0-SNAPSHOT"
+
+    //  Versions
+    static String awsBom = "2.5.29"
+    static String commonsIO = "2.6"
+    static String guava = "27.1-jre"
+    static String ingestAPI = "0.1.1"
```

Comment:
✏️ Maybe the API versions should be in their own group.

Reply:
I feel that's not really needed since they're just another dependency, but I don't feel too strongly on that